### PR TITLE
Fixed some buttons

### PR DESCRIPTION
**Pull request by apooravc**
There were some places where:-

1) some buttons were invisible in style_light theme:
    a) demographics menu "History | Report ..." tabs.
    b) fees> batch payments
2) "headers.inc.php" file or "cp-type" not present:
    a) Reports> Procedures > pending orders and procedure statistics
    b) Admin> Users> View facility specific information
    c) QA measures (view reports, create mips report, select measures, delete old reports, remove provider assignments, alerts log)

@teryhill @Trodrige Please have a look.

### CODE REVIEW

**Comment by teryhill** [comment]
just changes to the button class.

File: modules/MIPS/clinical_measures.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
 $fake_register_globals = false;
 
 require_once 'mips_headers.inc.php';
-
```
Comment (aethelwulffe) [inline modules/MIPS/clinical_measures.php]: Not needed here

File: modules/MIPS/measure_activate.php [hunk 2]
```diff
@@ -24,6 +24,7 @@
 $fake_register_globals = false;
 
 require_once '../../interface/globals.php';
+require_once '../../library/headers.inc.php';
 require_once $srcdir.'/api.inc';
```
Comment (aethelwulffe) [inline modules/MIPS/measure_activate.php]: Not here either.

File: modules/MIPS/report_results.php [hunk 3]
```diff
@@ -39,6 +39,7 @@
 //
 
 require_once 'mips_headers.inc.php';
+require_once '../../library/headers.inc.php';
 /** Current format of date  */
```
Comment (aethelwulffe) [inline modules/MIPS/report_results.php]: Not needed here either.

File: modules/MIPS/reset_provider_to_unassigned.php [hunk 4]
```diff
@@ -19,15 +19,16 @@
 require_once '../../interface/globals.php';
 include_once("$srcdir/api.inc");
 include_once("$srcdir/acl.inc");
+require_once '../../library/headers.inc.php';
```
Comment (aethelwulffe) [inline modules/MIPS/reset_provider_to_unassigned.php]: OR here.

File: modules/MIPS/truncate_reports.php [hunk 5]
```diff
@@ -3,29 +3,30 @@
...
 	 * @link    http://librehealth.io
 	 *
 	 * Please support this product by sharing your changes with the LibreHealth.io community.
-	 * 
+	 *
 	 * IMPORTANT:
 	 * This whole feature needs to be moved to a integrated admin form, and naturally use css buttons and translations.
 	 */
 	require_once '../../interface/globals.php';
+	require_once '../../library/headers.inc.php';
```
Comment (aethelwulffe) [inline modules/MIPS/truncate_reports.php]: Or here.

**Comment by aethelwulffe** [comment]
OK, if this helps buttons out, and the headers file is actually required to manage that, then I see the point.  I had a PR in against a lot of this, which is now merged.
Please rebase this, and then I will test.

**Comment by aethelwulffe** [comment]
I resolved conflict and version stuff, and will test the changes again.

**Comment by aethelwulffe** [comment]
I *think* everything still works.
If the MIPS stuff isn't working, then I will find out later :)

**Pull request merged**